lib/workers/repository/process/lookup/index.ts

```diff
@@ -372,6 +372,45 @@ export async function lookupUpdates(
       );
       let shrinkedViaVulnerability = false;
       if (config.isVulnerabilityAlert) {
+        if (config.vulnerabilityFixVersion) {
+          res.vulnerabilityFixVersion = config.vulnerabilityFixVersion;
+          if (versioning.isValid(config.vulnerabilityFixVersion)) {
+            // Filter out versions if the vulnerabilityFixVersion is higher
+            const filterCondition = (release: Release): boolean =>
+              versioning.isVersion(config.vulnerabilityFixVersion)
+                ? !versioning.isGreaterThan(
+                    config.vulnerabilityFixVersion!,
```

Comment:
```suggestion
                    config.vulnerabilityFixVersion,
```

this is already validated in if above or is typescript not able to follow 🤔

Reply:
TS doesn't follow it somehow